history/cultures/highborne.txt

```diff
@@ -28,6 +28,7 @@
 	discover_innovation = innovation_manorialism 
 	discover_innovation = innovation_baliffs 
 	#
+	discover_innovation = innovation_arid_warfare
```

Comment:
What? They live in Ferelas, jungles.

Reply:
should they get a the guerilla warfare bonus or do highborne not fight that way?

Reply:
> should they get a the guerilla warfare bonus or do highborne not fight that way?

It's not clear because they don't fight (they lose), but they may get it, yes.